src/components/metadata/styles/metadata-code.scss

```
@@ -71,19 +71,19 @@
 
 .pipeline-metadata-code__title {
   flex-grow: 0;
-  margin: 2.1em 36px 1.8em 36px;
+  font-size: 18px;
```

I would leave this in `em`s, as we generally try and use that unit for font-sizes. 

```suggestion
  font-size: 1.8em;
```